pkg/network/event_common.go

```diff
@@ -246,6 +246,12 @@ func (c ConnectionStats) ByteKey(buf []byte) ([]byte, error) {
 	return buf[:n], nil
 }
 
+// IsShortLived returns true when a connection went through its whole lifecycle
+// between two connection checks
+func (c ConnectionStats) IsShortLived() bool {
```

Comment:
The perf buffer is also used for UDP connections. Is there a way we can have a similar check for UDP connections, otherwise they won't benefit from this improvement.

Reply:
@p-lambert thoughts on UDP?

Reply:
Humm, ideally we want to have something in the UDP `ConnectionStats` that indicates its "provenance" (whether it came from the perf-ring or not) so we can apply the retry policy accordingly.
I really don't want to add a new field for that, so what I'd do is set `IPTranslation` to a sentinel value (like `RetryMe`) inside [this method](https://github.com/DataDog/datadog-agent/blob/pedro%2Fconntrack-race/pkg/network/tracer/tracer.go#L270)
WDYT? I guess the only drawback I see for this is that this would include every UDP connection read off the perf ring, and not necessarily only the short-lived ones.

Reply:
Let's give that a shot and see how that affects performance.